packages/storybook/stories/Components/Charts/DonutChart.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import { DonutChart } from "../..";
+import { createComponentStories } from "../../util";
+
+const { defaultExport, createStory } = createComponentStories({
+  component: DonutChart,
+  args: {
+    height: 300,
+    dataKey: "name",
+    category: "value",
+    showLegend: true,
+    showTooltip: true,
+    showAnimation: true,
```

Comment:
ain't these enabled by default?

Reply:
yes they are, but if we don't include them they don't show up in controls. This is due to the fact that I'm not going through `createBentoComponents` for Charts, so react-docgen does not work (see the comment in `stories/index.tsx` for more context)